UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -298,11 +304,16 @@ public enum BurnDamageLevels
 			CHARRED
 		}
 
-		public enum TramuticDamageTypes
+		[Flags]
+		public enum TramuticDamageTypes 
 		{
-			SLASH,
-			PIERCE,
-			BURN
+			SLASH  = 0,
+			PIERCE = 1,
```

Comment:
I think you Set up the flags wrong?
Apparently you can't use zero, 

wasn't this supposed to be integrated with the damage list on body parts?

Reply:
yeah, use the shifting technique to assign values, something like this:

````cs
[Flags]
public enum TramuticDamageTypes 
{
    None = 0,
    Slash = 1 << 0,
    Pierce = 2 << 1,
    Burn = 3 << 2
}
```

Also, why did you  create combined versions after Burn?

Reply:
> Also, why did you create combined versions after Burn?

Because for some reason unity doesn't allow you to combine two flags. not sure if it's a bug or what exactly but that's a workaround for it for now.

Reply:
combine where? In the dropdown in editor?

Reply:
Yep

Reply:
I think you have to use the naughty attributes flag

Reply:
Yep, Bod was right. It now works correctly and adjusted it to that.